### DISCUSSION TITLE
Can't open files/folders

### DISCUSSION THREAD

**Discussion by diogocccarvalho** [zed-industries/zed#15123] [Feature Requests]
### My Issue

Everytime I try to open a new project/add folder I get a:
![Screenshot_2024-07-24-22-34-55_1920x1080](https://github.com/user-attachments/assets/d0c46efd-443b-4a4b-9a7b-27b29dc340a0)

I'm using bspwm as my tiling window manager, and I guess it doesn't use XDG desktop portals.
The only thing I can find as a fix is:
![image](https://github.com/user-attachments/assets/3677939a-70d3-4367-b737-e972f3200bb4)

Which isn't at all what I'd consider a fix. Anyone have any ideas?
P.S. I'm not that well-versed at bspwm nor do I know the entire wiki info/edit the config files, etc...  so I'd need some step by step type stuff plz

**Comment by sleeping-at-night** [comment]
The link mentioned in the second image is a link to arch wiki about XDG Desktop Portal.
Which I indeed consider it as a fix in this situation.

If you using Arch based system, and wish to follow the said wiki page. I will summarize the step as following:
1. Install `xdg-desktop-portal`.
2. Choose one or more backend. In this case I will follow the wiki and choose lxqt backend and gtk backend.
3. Install the choosen backends. In this case install both `xdg-desktop-portal-lxqt` and `xdg-desktop-portal-gtk`.
4. Create file at `~/.config/xdg-desktop-portal/portals.conf` and put the following content inside:
```
[preferred]
default=gtk
org.freedesktop.impl.portal.FileChooser=lxqt
```

This should provide you with a solution for the file chooser problem.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
